documentation/analytics/recipes/player-geo-cluster-analysis.md

```diff
@@ -9,28 +9,28 @@ You will answer the following questions: How many players have played your game
 The coordinates associated with a given geohash can vary from median or random.
 For instance, take the following 5 coordinates within a geohash (square):
 
```

Comment:
Start and stop any section that includes ASCII Art with three (3) ` characters on a single line and end with the same after, and you don't have to add spaces to keep formatting.